### ISSUE TITLE
ODT tempalte doesn't show styles

### ISSUE THREAD

**Issue by raffaem** [jgm/pandoc#10327]
Following the documentation for `--reference-doc=` option available [here](https://pandoc.org/MANUAL.html#options-affecting-specific-writers), if I generate a docx template with:

```
pandoc -o custom-reference-doc.docx --print-default-data-file reference.docx
```

The resulting file will show the styles:

![image](https://github.com/user-attachments/assets/bcc8fbc0-bc4c-49ec-9280-a4880efb5f3f)

But if I create a ODT template with:
```
pandoc -o custom-reference-doc.odt --print-default-data-file reference.odt
```

the resulting file only contains "Hello world":

![image](https://github.com/user-attachments/assets/11c92566-d869-445e-b897-1f807a390eed)

Would it be useful to show the styles also in the ODT template?

Having said that, my understanding is that the contents get ignored in both cases and only the styles are applied.

**Comment by jgm** [comment]
> Would it be useful to show the styles also in the ODT template?

Yes, I'd say it would be!  It helps you see the effects of your style adjustments.

**Comment by raffaem** [comment]
Where are those `reference.docx` and `reference.odt` files?

I can't find them in the source tree :(

**Comment by tarleb** [comment]
Those files are generated from the contents of the `data/docx` and `dat/odt` directories, respectively:

E.g.

``` sh
cd data/docx && zip -r ../reference.docx *
```

should give you a reference file in `data/reference.docx`.

**Comment by raffaem** [comment]
I think I'll leave the task for someone more attuned of which styles are used where
